app/src/main/java/ch/epfl/cs311/wanderwave/ui/components/AppBottomBar.kt

```diff
@@ -45,34 +55,49 @@ fun AppBottomBar(navActions: NavigationActions) {
           Modifier.fillMaxWidth()
               .background(MaterialTheme.colorScheme.surface)
               .testTag("appBottomBar"),
-  ) {
-    // Assumes TOP_LEVEL_DESTINATIONS are in the order of Track List, Main, and Map for indexing
-    IconButton(
-        onClick = { navActions.navigateToTopLevel(TOP_LEVEL_DESTINATIONS[0].route) },
-        modifier =
-            Modifier.weight(1f) // Spread the icons evenly across the BottomAppBar
-                .testTag("bottomAppBarButton" + Route.TRACK_LIST.routeString)) {
-          Icon(
-              trackListIcon,
-              contentDescription = stringResource(id = TOP_LEVEL_DESTINATIONS[0].textId))
-        }
+      containerColor = Color.Black) {
+        // Assumes TOP_LEVEL_DESTINATIONS are in the order of Track List, Main, and Map for indexing
+        IconButton(
+            onClick = { navActions.navigateToTopLevel(TOP_LEVEL_DESTINATIONS[0].route) },
+            modifier =
+                Modifier.weight(1f) // Spread the icons evenly across the BottomAppBar
+                    .testTag("bottomAppBarButton" + Route.TRACK_LIST.routeString)) {
+              Column(
+                  verticalArrangement = Arrangement.Center,
+                  horizontalAlignment = Alignment.CenterHorizontally,
+                  modifier = Modifier.fillMaxHeight()) {
+                    Icon(
+                        trackListIcon,
+                        contentDescription = stringResource(id = TOP_LEVEL_DESTINATIONS[0].textId),
+                    )
+                    Text(text = "Track List", style = MaterialTheme.typography.bodySmall)
+                  }
+            }
 
-    Spacer(Modifier.weight(1f)) // Spacer for centering the middle icon
+        IconButton(
+            onClick = { navActions.navigateToTopLevel(TOP_LEVEL_DESTINATIONS[1].route) },
+            modifier = Modifier.weight(1f).testTag("bottomAppBarButton" + Route.MAP.routeString)) {
+              Image(
+                  modifier = Modifier.weight(4f).padding(top = 5.dp, bottom = 5.dp),
+                  painter = painterResource(id = R.drawable.beaconlogo),
```

Comment:
I like the beacon logo here instead of the default map marker. Is there maybe a way to make it clearer that this is the map screen? Maybe a map icon with the beacon logo on it?

Reply:
I think it is somehow easily understood once the user sees beacons on the map.

Reply:
But I will try to come up with something that doesn't look too crowded

Reply:
thanks

Reply:
![image](https://github.com/G12-Wanderwave/Wanderwave/assets/84101643/e888725a-5ecf-40a2-9467-2385f2ce0dc1)
turns out the layout doesn't allow me to have vertical space and overlapping makes it look bad. I propose this version lmk what you think